<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newWindow</class>
 <widget class="QDialog" name="newWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E2DED3;
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: black;
font-weight: bold</string>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: #472FF7</string>
      </property>
      <item>
       <property name="text">
        <string>Select Category</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shopping</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vehicle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LifeStyle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clothing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Investments</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Miscellaneous</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poor Richard</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #472FF7;
letter-spacing: 10px</string>
   </property>
   <property name="text">
    <string>New Entry</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-weight: bold</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: black;
font-weight: bold</string>
      </property>
      <property name="text">
       <string>Amount(in â‚¹)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="amountInput">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: black;
font-weight: bold</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">color: #472FF7;
background: white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-weight: bold</string>
   </property>
   <property name="text">
    <string>Payment Method</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>355</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="submitBtn">
      <property name="styleSheet">
       <string notr="true">color: #FF6D24;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">color: #5D12E8;
font-weight: bold</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="paymentMethodInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;
background: white;</string>
   </property>
   <item>
    <property name="text">
     <string>Choose payment method</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UPI</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
